### PR TITLE
Crab Rockets

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
add: Look sir, free crabs!
/:cl:

Crab Rockets are now orderable as a drop pod only purchase.

### CODE REVIEW

**Comment by Buggy123** [comment]
Y E S

**Comment by Chronitonity** [comment]
look sir free crabs

**Comment by optimumtact** [comment]
goof if you pull a stunt like this again I will disqualify you.

This is you first and only warning

**Comment by Buggy123** [comment]
REE

**Comment by ike709** [comment]
citrus tyranny

**Comment by MattDuskson** [comment]
YOU HAVE MY VOTE

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1136,6 +1136,18 @@
...
+/datum/supply_pack/organic/critter/crab
+	name = "Crab Rocket"
+	cost = 5000
+	contains = list(/mob/living/simple_animal/crab)
+	crate_name = "look sir free crabs"
+	DropPodOnly = TRUE
+
+/datum/supply_pack/organic/critter/crab/generate()
+	. = ..()
+	for(var/i in 1 to 49)
```
Comment (81Denton) [inline code/modules/cargo/packs.dm]: Is the gin messing with me or does this have the chance to spawn 49 crabs

Comment (Iamgoofball) [reply code/modules/cargo/packs.dm]: crab rocket

Comment (MrStonedOne) [reply code/modules/cargo/packs.dm]: not chance, it always spawns 49 crabs

Comment (81Denton) [reply code/modules/cargo/packs.dm]: This is fine

**Comment by Xhuis** [comment]
`Iamgoofball  wants to merge 1 commit into tgstation:master from Iamgoofball:vote-goof`

**Comment by ike709** [comment]
Goof wouldn't be the worst person to win an election.

**Comment by Buggy123** [comment]
CRAB.
ROCKET.

**Comment by SyruseEln** [comment]
Speedmerge please.

**Comment by MrDoomBringer** [comment]
i massively approve of this please do it for cargonia

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -1136,6 +1136,18 @@
 	for(var/i in 1 to 49)
 		new /mob/living/simple_animal/butterfly(.)
 
+/datum/supply_pack/organic/critter/crab
+	name = "Crab Rocket"
+	cost = 5000
+	contains = list(/mob/living/simple_animal/crab)
+	crate_name = "look sir free crabs"
```
Comment (LetterN) [inline code/modules/cargo/packs.dm]: >free crabs
>`cost = 5000`
unplayable.

Comment (PKPenguin321) [reply code/modules/cargo/packs.dm]: the rocket is the part that costs money

Comment (Iamgoofball) [reply code/modules/cargo/packs.dm]: ^

Comment (factoryman942) [reply code/modules/cargo/packs.dm]: crates are worth 500
correct manifests are worth 200
make it cost 700 credits to order

also who puts their crabs in a rocket? Drop crate or bust

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/cargo/packs.dm]: actually it always spawns 50 crabs because it contains one other crab in the contents

**Comment by Noober84555** [comment]
i approve this

**Comment by outbools** [comment]
Everyone against it is Ravenholm residents 
image

**Comment by ZeWaka** [comment]
a good branch name

**Comment by Iamgoofball** [comment]
@optimumtact do you want me to change this over to a new branch too btw, this was from before the advertising rule got put into place

**Comment by ExcessiveUseOfCobblestone** [comment]
No one reads branch names!

**Comment by Buggy123** [comment]
C R A B R O C K E T S
R A B R O C K E T S
A B R O C K E T S
B R O C K E TS
R O C K E T S
O C K E T S
C K E T S
K E T S
E T S
T S
S

**Comment by deranging** [comment]
@outbools bitch that's a picture of the last fucking level from route kanal (d1_canals_05) not ravenholm get your facts right

**Pull request merged**